lib/momento/response/sorted_set/sorted_set_fetch_response.rb

```diff
@@ -0,0 +1,104 @@
+require 'grpc'
+require_relative '../error'
+require_relative '../../generated/cacheclient_pb'
+
+module Momento
+  # A response containing the elements retrieved from a sorted set.
+  class SortedSetFetchResponse < Response
+    # The sorted set exists and any matching elements were fetched.
+    # @return [Boolean]
+    def hit?
+      false
+    end
+
+    # The sorted set does not exist.
+    # @return [Boolean]
+    def miss?
+      false
+    end
+
+    # The fetched values as UTF-8 Strings and their scores.
+    # @return [(Array[{ value: String, score: Float }] | nil)] the UTF-8 elements and their scores
+    def value_string_elements
```

Comment:
can we add an alias that is just `value` that defaults to returning `value_string_elements`?

Reply:
Yeah. I'll add it in a follow on pr.